codespell_lib/data/dictionary.txt

```diff
@@ -2867,10 +2881,12 @@ colomns->columns
 colon-seperated->colon-separated
 colonizators->colonizers
 colorfull->colorful
+colorizoer->colorizer
 coloum->column
 coloumn->column
 coloumns->columns
 coloums->columns
+colours->colors
```

Comment:
remove this because this is correct british spelling

Reply:
Done